rails/background_jobs.md

```diff
@@ -141,19 +147,25 @@ end
 ```
 
 2) As an environment variable
-```
-ENV['GOOD_JOB_QUEUES'] = mailers:1;default:3;pdf:1
+```ruby
+ENV['GOOD_JOB_QUEUES'] = 'mailers:1;default:3;pdf:1;-pdf'
 ```
 
 ### Job retries
-We want to ensure that jobs which fail for transient reasons (like a network issue) are retried automatically. For this reason jobs should be written in a way that is idempotent (able to be run multiple times and produce the same result). We also want to ensure that our job system is configured to automatically retry failed jobs by default. Because libraries like noticed or action mailer do not inherit from your app’s ApplicationJob class it is recommended to include the following configuration in an initializer:
+We want to ensure that jobs which fail for transient reasons (like a network issue) are retried automatically. For this reason jobs should be written in a way that is idempotent (able to be run multiple times and produce the same result). We also want to ensure that our job system is configured to automatically retry failed jobs by default. Because libraries like `noticed` or Action Mailer do not inherit from your app’s ApplicationJob class, it is recommended to include the following configuration in an initializer (order is important, see `rolemodel_rails#good_job` generator):
```

Comment:
Should `noticed` be Noticed? That way it will match the format of Action Mailer.

Reply:
I think [ActionMailer is a library](https://github.com/rails/rails/tree/adf0c733399d5d410c61cc423941c99846604676?tab=readme-ov-file#frameworks-and-libraries), as is Noticed. The classes we care about in this case would be `ActionMailer::Base`, and `Noticed::ApplicationJob`.